src/SFML.Window/ContextSettings.cs

```diff
@@ -112,7 +113,10 @@ public override string ToString()
         /// <summary>The attribute flags to create the context with</summary>
         public Attribute AttributeFlags;
 
+        /// <summary>Internal Representation</summary>
+        private int _SRgbCapable;
+
         /// <summary>Whether the context framebuffer is sRGB capable</summary>
-        public bool SRgbCapable;
+        public bool sRGBCapable { get => _SRgbCapable == 0; }
```

Comment:
Accidental value flip? shouldn't it be `_SRgbCapable == 1;`?

Reply:
You, you're right.  I stared at different boolean types for far too long.